erpnext/utilities/page/leaderboard/leaderboard.js

```diff
@@ -141,7 +142,7 @@ frappe.Leaderboard = Class.extend({
 						],
 						labels: graph_items.map(d=>d.name)
 					},
-					colors: ['light-green'],
+					colors: ['orange'],
```

Comment:
green is success, orange is warning - lets go with green 👍

Reply:
:+1: